scripts/flash.sh

```diff
@@ -52,8 +52,6 @@ function usage {
         printf "\t$type\n"
     done
     printf "\n"
-    
-    printf "Remember that you need to install arduino-cli first before you can use this flash script!\n\n"
     exit 1
```

Comment:
Why did we delete this message?

Reply:
It is already printed at the bottom if the script can't find `arduino-cli` on the path

Reply:
Oh okay I didn't realize that, that makes sense!